### ISSUE TITLE
[SampleCard-FlightDetails.json]: Screen reader announcement is not descriptive enough for '4 hours 15 minutes' to express the objective.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#90]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**
1. Turn on the Talkback.
2. Navigate to the Visual.
3. Navigate to the sample card.
4. Using swipe gesture, swipe left to right in the card.
5. Navigate to "4hours 15 minutes" text and observe the issue. 

**Actual Result:**
Screen reader announcement is not descriptive enough for '4 hours 15 minutes' to express the objective.
Screen reader only announces '4 hours 15 minutes' when focus moves on the said text object.

**Expected Result:**
Screen reader announcement should be descriptive enough and be sufficient to express the purpose ~ Text block->"4 hours 15 minutes" like time duration/delay.

**User Experience:**
The screen reader users will get impact and might get confuse when there is no label info provided for text block element describing its purpose.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/9329e2de-a84c-46e7-9b6a-8ae1ca48c2e4

![Screen reader announcement is not descriptive enough for '4 hours 15 minutes' to express the objective](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/721571ec-0210-467a-93e9-a345212f9b6b)

### Card JSON

```JSON
NA
```
